.woodpecker/docker.yml

```diff
@@ -68,13 +68,38 @@ pipeline:
       branch: ${CI_REPO_DEFAULT_BRANCH}
       event: push
 
+  publish-branch-server:
```

Comment:
What do you think about naming this `publish-stable-server` or `publish-release-branch-server` or something different? The meaning of `branch` wasn't directly clear to me. Maybe a comment would be an alternative, if you want to stick with `publish-branch-server`.

Reply:
this are just pipeline step names ... if you think us maintainers wont understand them - just propose a new name ...

Reply:
Okay, lets use: `publish-release-branch-server`

Reply:
done